admin/db.php

```diff
@@ -314,7 +314,7 @@ function gigpress_db_upgrade_160() {
 		}
 	}
 
-	// Try our darndest to extract states from cities and put them in their own column
+	// Try our darnedest to extract states from cities and put them in their own column
```

Comment:
this isn't a typo
![image](https://user-images.githubusercontent.com/1812179/77480153-301c6500-6def-11ea-9802-91fee91feed9.png)

However, we could use less colloquial wording, such as "Try our hardest..."

Reply:
```suggestion
	// Try our hardest to extract states from cities and put them in their own column
```